docs/source/apidoc/core.rst

```diff
@@ -114,6 +126,9 @@ Available Channels
    :members:
    :show-inheritance:
 
+.. automodule:: pulser.channels.dmm
```

Comment:
Maybe here we could make a subsection for the DMM, even change the docstring at the top of the file to be more informative. As it is, it appears a bit out of the blue.

Reply:
Ah okay I see the issue. I still think that the dmm has its place here in the available channels. I suggest to have an autoclass instead an automodule. What do you think of this ?

Reply:
Should be fine, yes. Just make sure the description in the class makes it clear it's not exactly a channel like the others